lumigator/jobs/inference/definition.py

```diff
@@ -0,0 +1,84 @@
+from uuid import UUID
+
+from inference.schemas import (
+    DatasetConfig,
+    HuggingFacePipelineConfig,
+    InferenceJobConfig,
+    InferenceServerConfig,
+    JobConfig,
+    SamplingParameters,
+)
+from lumigator_schemas.jobs import JobCreate, JobType
+
+from backend.services.exceptions.job_exceptions import JobValidationError
+from backend.services.job_interface import JobDefinition
+
+# Served models
+OAI_API_URL: str = "https://api.openai.com/v1"
+MISTRAL_API_URL: str = "https://api.mistral.ai/v1"
+DEEPSEEK_API_URL: str = "https://api.deepseek.com/v1"
```

Comment:
We don't need this anymore. Some of this code looks like it's pre LiteLLM and some of Hareesh's hanges, may be worth double checking merges 😅 
```suggestion
```

Reply:
It's complicated to keep track of things that get moved _and_ modified :-/ I really don't know how to best handle these situations. Thanks a lot for the extra pair of eyes checking these issues.

Reply:
I'll check the definitions according to the latest main. If you spot something else, please let me know.

Reply:
With the codebase being relatively new and small, things are changing super fast, so I think the best way to avoid this is by having small PRs that get opened and merged quickly. The longer they sit open, the more they risk becoming out of date. I think this will be less of a problem as the codebase grows and the structure stabilizes.

Reply:
Furthermore, I'll wait for @HareeshBahuleyan 's PR on translation and then sync the job definitions.

Reply:
Ok, diff'ed with main and manually ported from main. It should be consistent but please give it a quick glance.